hackathon_site/review/forms.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+
+from django import forms
+
+
+class MailerForm(forms.Form):
+    date_start = forms.DateField(label="Start date", initial=datetime.date.today)
```

Comment:
Could you add `widget=forms.DateInput(format="%Y-%m-%d", attrs={"type": "date"})` to both of these date fields?

On browsers that support HTML5 date inputs (type="date"), this will render a date picker.